projects/igniteui-angular/src/lib/grids/grid/grid-filtering-ui.spec.ts

```diff
@@ -3474,6 +3443,84 @@ describe('IgxGrid - Filtering Row UI actions #grid', () => {
             const headerChip = GridFunctions.getFilterChipsForColumn('ProductName', fix);
             expect(headerChip.length).toBe(1);
         }));
+
+        it('Should commit the input and new chip after focus out and should edit chip without creating new one.', fakeAsync(() => {
+            // Click date filter chip to show filter row.
+            const filterCells = fix.debugElement.queryAll(By.directive(IgxGridFilteringCellComponent));
+            const dateFilterCell = filterCells.find((fc) => fc.componentInstance.column.field === 'ReleaseDate');
+            const dateFilterCellChip = dateFilterCell.query(By.directive(IgxChipComponent));
+            dateFilterCellChip.nativeElement.click();
+            tick(100);
+            fix.detectChanges();
+
+            // Click input to open calendar.
+            const filteringRow = fix.debugElement.query(By.directive(IgxGridFilteringRowComponent));
+            const input = filteringRow.query(By.directive(IgxInputDirective));
+            input.nativeElement.click();
+            tick(100);
+            fix.detectChanges();
+
+            // Click the today date.
+            const outlet = document.getElementsByClassName('igx-grid__outlet')[0];
+            let calendar = outlet.getElementsByClassName('igx-calendar')[0];
+            const todayDayItem = calendar.querySelector('.igx-calendar__date--current');
+            (<HTMLElement>todayDayItem).click();
+            tick(100);
+            fix.detectChanges();
+
+            // Focus out
+            clickElemAndBlur(dateFilterCellChip, input);
+            tick(200);
+            fix.detectChanges();
+            expect(dateFilterCellChip.componentInstance.selected).toBeFalsy('initial chip is not committed');
+            expect((<IgxInputDirective>input.componentInstance).value).toBeNull('initial input value is present and not committed');
```

Comment:
`dateFilterCellChip` is the filter cell chip, you should use the condition cell chip instead.

Reply:
You should still replace the `dateFilterCellChip` with the condition chip in the `clickElemAndBlur` call.